vspace/src/main/java/edu/asu/diging/vspace/web/staff/ListImagesController.java

```diff
@@ -30,25 +30,32 @@
     private ISpaceManager spaceManager;
 
     @RequestMapping("/staff/images/list")
-    public String listSpacesWithoutNum(Model model) {
-        return "redirect:/staff/images/list/1";
+    public String listSpacesWithoutNum(Model model, @RequestParam(value = "imageCat", required = false) String imageCategory) {
+
+        return String.format("redirect:/staff/images/list/1?imageCat=%s",(imageCategory==null?"":imageCategory));
     }
 
     @RequestMapping("/staff/images/list/{page}")
     public String listSpaces(@PathVariable(required = false) String page,
+            @RequestParam(value = "imageCat", required = false) String imageCategory,
             @RequestParam(value = "sort", required = false) String sortedBy,
             @RequestParam(value = "order", required = false) String order, Model model) {
         int pageNo;
         page = StringUtils.isEmpty(page) ? "1" : page;
+        ImageCategory category = null;
+        if(!imageCategory.isEmpty()) {
+            category = ImageCategory.valueOf(imageCategory);
```

Comment:
I believe this line can throw a `IllegalArgumentException`. Since this is unvalidated input from the front end you should handle this here.